compiler/testData/codegen/boxAgainstJava/sam/differentFqNames.kt

```diff
@@ -1,5 +1,3 @@
-// IGNORE_BACKEND: JVM_IR
-// WITH_RUNTIME
```

Comment:
Please return `WITH_RUNTIME` in these tests:
* `boxAgainstJava/sam/differentFqNames.kt`
* `boxAgainstJava/sam/samConstructorGenericSignature.kt`
* `boxAgainstJava/sam/adapters/inheritedOverriddenAdapter.kt`

(see cb7727d51a)

Reply:
Whoops, should've paid more attention when rebasing, sorry!